src/fal/utils.py

```diff
@@ -59,10 +59,10 @@ def cache_static(func):
     _function_cache = _NO_HIT
 
     @functools.wraps(func)
-    def wrapper():
+    def wrapper(*func_args, **func_kwargs):
```

Comment:
Can you also change the docstring to reflect this?

Reply:
actually changed it to import a backported version